<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeightSelect</class>
 <widget class="QWidget" name="HeightSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HeightSelect</string>
  </property>
  <widget class="QGraphicsView" name="framePreview">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="frameBack">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="frameForward">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="singleLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="doubleLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="singleLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Single box height (px)</string>
   </property>
  </widget>
  <widget class="QLabel" name="doubleLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Double box height (px)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="autoLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>auto detect box heights</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>510</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
